samcli/lib/pipeline/bootstrap/stage.py

```diff
@@ -290,3 +289,8 @@ def print_resources_summary(self) -> None:
             )
             click.secho(f"\tACCESS_KEY_ID: {self.pipeline_user.access_key_id}", fg="green")
             click.secho(f"\tSECRET_ACCESS_KEY: {self.pipeline_user.secret_access_key}", fg="green")
+
+    @staticmethod
```

Comment:
should we make it a non static method as we user `self.name` as part of the stack-name by design

Reply:
changed